PasswordVerifier.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace PasswordVerifier
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter your password: ");
+            string password;
+            password = Console.ReadLine();
+            bool isTrue = true;
+            try
+            {
+                if (string.IsNullOrWhiteSpace(password))
+                {
+                    throw new Exception("Password should not be empty");
+                    isTrue = false;
```

Comment:
This will always set isTrue as true, set this value before throwing exception.

Reply:
Resolved on the new commit. Thank you!